guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+      alt: Federation contact information display in a wallet interface
+      caption: Contact information is optional for the federation operator to include. If there is no contact information, consider not displaying this metadata.
+    - file: federation-id4
+      modalImage: federation-id4@2x
+      alt: Example of important announcements display in a wallet interface
+      caption: A message of the day can consists of anything the federation operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance.
+
+images_cashu-id:
+    - file: cashu-id1
+      modalImage: cashu-id1@2x
+      alt: Mint identity and version display in a wallet interface
+      caption: Consider the most important information to display to the user about the mint.
+    - file: cashu-id2
+      modalImage: cashu-id2@2x
+      alt: Mint descriptions display in a wallet interface
+      caption: Different mint operator may use the description fields differently. Consider this when designing your mint details screen.
+    - file: cashu-id3
+      modalImage: cashu-id3@2x
+      alt: Mint contact information display in a wallet interface
+      caption: Contact information is optional for the mint operator to include. If there is no contact information, consider not displaying this metadata.
+    - file: cashu-id4
+      modalImage: cashu-id4@2x
+      alt: Example of important announcements display in a wallet interface
+      caption: A message of the day can consists of anything the mint operator wants to tell the user. It can for example be used to announce a new feature or a upcoming maintenance.
+    - file: cashu-id5
+      modalImage: cashu-id5@2x
+      alt: Example of supported features display in a wallet interface
+      caption: Supported features are pulled by indexing a mints information and reading what nuts the mint has implemented.
+
+---
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/ecash/best-practicies.jpg"
+   retina = "/assets/images/guide/how-it-works/ecash/best-practicies@2x.jpg"
+   mobile = "/assets/images/guide/how-it-works/ecash/best-practicies-mobile.jpg"
+   mobileRetina = "/assets/images/guide/how-it-works/ecash/best-practicies-mobile@2x.jpg"
+   alt-text = "ecash best practices illustration"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+<!--
+
+
+This is a summary page of best design practices for both bitcoin-backed ecash protocols: Cashu and Fedimint.
+
+The design source for screen mock-ups on this page can be found here:
+ADD Figma URL
+
+
+-->
+
+# Ecash design best practices
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+## General Ecash Best Practices
+When designing bitcoin-backed ecash applications it's important to prioritize clear and intuitive interfaces that allow users to easily manage their tokens, whether they are minting, sending, or redeeming them.
```

Comment:
<img width="836" alt="image" src="https://github.com/user-attachments/assets/a2ee7202-cb20-4889-9c6d-6e0b585ff96f">I want to include something more explicit that mentions the sending practices are the same as those in the existing reference wallet design. Guidelines like entering an amount, displaying fees, and showing a success message still apply. However the key difference is that ecash wallets allow users to generate a payment in the form of an ecash token, which can be shared as a QR code or a string of text. It might be enough to mention this in the text, though I could include an image of a QR code with its text string underneath to show an example. But this image is already shown in the pending tokens section right below.